Realm/RLMUtil.mm

```diff
@@ -280,7 +280,8 @@ static inline BOOL RLMIsSubclass(Class class1, Class class2) {
 }
 
 BOOL RLMIsObjectSubclass(Class klass) {
-    return RLMIsSubclass(class_getSuperclass(klass), RLMObjectBase.class);
+    static Class objectBaseClass = RLMObjectBase.class;
```

Comment:
Ah gotcha. Might it make sense to have a global static for the object base class that's done inside `+initialize` somewhere?